csharp/extractor/Semmle.Extraction.CSharp/Entities/Accessor.cs

```diff
@@ -30,6 +30,10 @@ protected Accessor(Context cx, IMethodSymbol init, IPropertySymbol property)
             return props.SingleOrDefault();
         }
 
+        public override bool NeedsPopulation =>
+            base.NeedsPopulation &&
+            !Symbol.IsPartialDefinition; // Accessors always have an implementing declaration as well.
```

Comment:
I don't think that aligns with how we extract partial methods; for partial methods we still only extract one entity, but it will have multiple locations.

Reply:
Not sure that I understand the comparison to partial methods. In this case we don't want to extract the methodsymbol used in the declaring defintion (as this is not a callable).

Reply:
But I think we may still want to extract it's location, like we do for partial methods.

Reply:
I think that the current implementation mirrors that of partial methods. For methods we might extract multiple method symbols (one for the declaration and one of the implementation), but both method symbols will (only) extract the implementation location as the source location (note the overrides of `BodyDeclaringSymbol` and `ReportingLocation` in `OrdinaryMethod`).

Reply:
Ok, so I double checked, and you are right, sorry for the confusion. We do indeed only extract one location for partial methods, it only for partial _classes_ that we extract multiple locations.

Reply:
Yes, I agree.